### ISSUE TITLE
wrgl pull 

### ISSUE THREAD

**Issue by ayyubibrahimi** [wrgl/wrgl#9]
Hey Khoi,

I ran into an issue with installing wrgl through Docker yesterday. Please see the error below and my solution (I'm running this on an M3 Pro). However, by making this change, I believe that I've introduced the `wrgl pull` error that was returned in the GH action for [this](https://github.com/ipno-llead/processing/actions/runs/7715803749/job/21031165637?pr=563) PR. Please advise. Thank you!

<img width="1506" alt="Screenshot 2024-01-30 at 4 28 48 PM" src="https://github.com/wrgl/wrgl/assets/57520778/b87d5a44-2a37-48e3-8603-641b94a06fc6">

**Comment by pckhoi** [comment]
What is the output of `uname` and `uname -m` on your M3?

**Comment by ayyubibrahimi** [comment]
uname: Darwin
uname -m: arm64

**Comment by pckhoi** [comment]
I'm sorry, but it looks like you were running those commands while building a Docker container. What was the error before you modified those lines?

**Comment by ayyubibrahimi** [comment]
Yes. I was running those commands while following the installation instructions from the `readme` in the gh_k8s_runner directory. The error that I received prior was related to the persistent storage issue with dvc pull. I ran the installation process in order to update the storage space.
